README.md

```diff
@@ -298,6 +302,7 @@ The theme allows configuration of specific pages via [front matter variables](ht
 The following variables are currently supported:
 
 * `tools_menu (optional): Boolean (shows the vertical tools menu on any page using the default template)`
+* `blog_card (optional): Boolean (shows the blog card on the home page)`
```

Comment:
It looks like there may have been a bad merge here. This and the previous line are completely between the ` characters instead of just the variables at the beginning of the lines.